src/RedArrow.Argo.Client/Linq/Queryables/WhereQueryable.cs

```diff
@@ -204,12 +204,12 @@ private string TranslateBinaryExpression(Expression expression)
             var bExpression = expression as BinaryExpression;
             if (bExpression == null) throw new NotSupportedException();
 
-            if (bExpression.NodeType == ExpressionType.AndAlso)
+            if (bExpression.NodeType == ExpressionType.AndAlso || bExpression.NodeType == ExpressionType.And)
```

Comment:
You must not have pulled latest.  I reverted this.  Found that NodeType.And and NodeType.Or are for BITWISE And and Or, respectively ;)